src/test/java/io/prestosql/plugin/tiledb/TestTileDBQueries.java

```diff
@@ -837,6 +865,16 @@ private void create1DVectorStringDimensionWithoutURI(String arrayName)
         queryRunner.execute(createSql);
     }
 
+    private void create1DVectorStringWithFilters(String arrayName)
+    {
+        QueryRunner queryRunner = getQueryRunner();
+        String createSql = format("CREATE TABLE %s(" +
+                "x varchar WITH (dimension=true,filter_list='(byteshuffle,-1), (gzip,9)'), " +
```

Comment:
byteshuffle/bitshuffle/RLE don't take parameters. Is it possible to support a filter without a numeric parameter?

Reply:
It is, it's at this part of the code: https://github.com/TileDB-Inc/TileDB-Presto/blob/c81646c7faa856eb7a123a14ce51f3e8bbda851a/src/main/java/io/prestosql/plugin/tiledb/util/Util.java#L278

In case that the filter does not take parameters we just ignore the filter parameter. So providing a parameter does not cause an error, but I corrected the unit test as well.